keyboards/split75/keymaps/art/keymap.c

```diff
@@ -0,0 +1,147 @@
+  * |     |     |  |     |     |     |     |     |     |     |     |     |     |     |     |     |     |           |     |
+  * |-----+-----|  |-----------------------------------------'  ,--------------------------------------------------+-----|
+  * |     |     |  |       |     |     |     |     |     |     |     |     |     |     |     |     |     |         |     |
+  * |-----+-----|  |---------------------------------------.    `--------------------------------------------------+-----|
+  * |     |     |  |         |     |     |     |     |     |     |     |     |     |     |     |     |             |     |
+  * |-----+-----|  |-----------------------------------------.   `-------------------------------------------------+-----|
+  * |     |     |  |           |     |     | Play|     |     |     |     |     |     |     |       |         |     |     |
+  * |-----+-----|  |-----------------------------------------'   ,-------------------------------------------+-----+-----|
+  * |     |     |  | BASE  |      |      |           |     |     |               |       |     |       |     |     |     |
+  * `-----------'  `---------------------------------------'     `-------------------------------------------------------'
+  */
+  [MEDIA] = KEYMAP(
+//--------------------------------Left Hand-----------------------------------------------| |--------------------------------Right Hand------------------------------------------------
+                      _______,  _______,  _______,  _______,  _______,  _______,  _______,              _______,   _______,  _______,  _______,  _______,  _______,  _______,  _______, _______, \
+_______,  _______,    _______,  _______,  _______,  _______,  _______,  _______,  _______,              _______,   _______,  _______,  _______,  _______,  _______,  _______,           _______, \
+_______,  _______,    _______,  KC_MPRV,  KC_VOLU,  KC_MNXT,  _______,  _______,              _______,  _______,   _______,  _______,  _______,  _______,  _______,  _______,           _______, \
+_______,  _______,    _______,  _______,  KC_VOLD,  _______,  _______,  _______,              _______,  _______,   _______,  _______,  _______,  _______,  _______,                     _______, \
+_______,  _______,    _______,  _______,  KC_MUTE,  KC_MPLY,  _______,  _______,              _______,  _______,   _______,  _______,  _______,                      _______, _______,  _______, \
+_______,  _______,    _______,  _______,  _______,  _______,  _______,                        _______,             _______, _______,   _______,                      _______, _______,  _______ \
+  )
+};
+
+// Runs constantly in the background, in a loop.
+void matrix_scan_user(void) {
+	uint8_t layer = biton32(layer_state);
+
+    led_1_off();
+    led_3_off();
+    switch (layer) {
+        case BASE:
+            led_3_on();
+            break;
+        default:
+            break;
+    }
+};
```

Comment:
This should use <https://docs.qmk.fm/#/custom_quantum_functions?id=layer-change-code>
Something like
```suggestion
layer_state_t layer_state_set_user(layer_state_t state) {
    led_1_off();
    led_3_off();
    switch (get_highest_layer(state)) {
    case BASE:
        led_3_on();
        break;
    }
    return state;
}
```